Basic-Car-Maintenance/Shared/Onboarding/Views/WelcomeViewAddVehicle.swift

```diff
@@ -9,99 +9,177 @@ import SwiftUI
 
 struct WelcomeViewAddVehicle: View {
     
-    // Logic to remember Onboarding screen to not load again when app is launched
-//    @AppStorage("isFirstTime") private var isFirstTime: Bool = true
+    // Logic to remember OnboardingScreen to not load again when app is launched
+    @AppStorage("isFirstTime") private var isFirstTime: Bool = true
+    @AppStorage("vehicleName") private var vehicleName: String = ""
+    @AppStorage("vehicleMake") private var vehicleMake: String = ""
+    @AppStorage("vehicleModel") private var vehicleModel: String = ""
     @Environment(\.dismiss) var dismiss
-    
-    @State private var vehicleName: String = ""
-    @State private var vehicleMake: String = ""
-    @State private var vehicleModel: String = ""
+    @State private var validationAlertName: Bool = false
+    @State private var validationAlertMake: Bool = false
+    @State private var validationAlertModel: Bool = false
+    @State private var onboardingViewCompletedAlert: Bool = false
```

Comment:
You can accomplish this with only one alert, and an optional error property with the different alert types

Reply:
Ok, I will do it 👍